### ISSUE TITLE
TODO: Get the training pipeline working on your machine

### ISSUE THREAD

**Issue by Json-To-String** [ssc40/MLharvest#27]
- [x] Run `format_filenames.py` (make sure the directory it modifies is correct)
- [x] Run `train_ndvi_unet.py`.
- [x] If that works, see if you can change the hyperparameters to push the performance higher. 

One thing is to ensure you have the correct version of pytorch (you most likely have it installed, but it needs to be GPU compatible. You can ensure this by making sure the `pyproject.toml` in your root directory is up to date and points to a cuda-compatible pytorch installation.)
Then run `uv sync --all-extras`

**Comment by Json-To-String** [comment]
Can you achieve a stable loss between train/valid within the first 10 epochs?

**Comment by Json-To-String** [comment]
This is done with our investigation
